### ISSUE TITLE
"The application to execute does not exist"

### ISSUE THREAD

**Issue by martinboers** [PLCnext/PLCnext_CLI_VS#4]
When attempting to create a new project in Visual Studio using a PLCnext template, an error message appears similar to the following:

```text
The application to execute does not exist: 'C:\Users\...\AppData\Local\.net\plcncli\...\plcnli.dll'
```

or 

```text
The application to execute does not exist: 'C:\Users\...\AppData\Local\Temp\.net\EngineeringLibraryBuilder\...\EngineeringLibraryBuilder.dll'.
```

I am aware of the cause (a caching issue with .NET Core 3.x) and the work-around (delete the plcncli or the EngineeringLibraryBuilder directory in the .NET directory).

This issue is a hint for others who experience a similar problem. The issue can be closed when this problem is resolved.

(internal reference: #208771)

**Comment by martinboers** [comment]
The following symptom is also related to this issue.

When trying to create a new project in Visual Studio, an error like the following may appear:

![image](https://user-images.githubusercontent.com/13133969/114826958-647f0c80-9dc8-11eb-8488-4272c41e39fd.png)

**Comment by martinboers** [comment]
In all cases the work-around is to delete the offending directory from the local .NET cache - for example, delete this directory and its contents:

`%temp%\.net\plcncli`

... which usually translates to:

`C:\Users\...\AppData\Local\Temp\.net\plcncli`

(substituting the elipsis with the relevant user name).

**Comment by mluhmann** [comment]
Link to the original issue:
https://github.com/dotnet/runtime/issues/3830

With the next version our CLI will switch to .NET 5.x. In the mean time you could set the `DOTNET_BUNDLE_EXTRACT_BASE_DIR` to a different, non temporary location as a workaround.

**Comment by OWarneke** [comment]
Error:  An assembly specified in the application dependencies manifest (plcncli.deps.json) was not found: package: 'FeatureToggle.Common', version: '4.0.2' path: 'lib/netstandard1.4/FeatureToggle.Common.dll'

**Comment by mluhmann** [comment]
Fixed with Version 2021.6
